cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/graphqlHelper.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { objectQuery } from 'services/helpers';
+import { GLOBALS } from 'services/global-constants';
+
+function getProgram(pipeline) {
+  const programs = objectQuery(pipeline, 'applicationDetail', 'programs');
+
+  if (programs === null || programs === undefined) {
+    return [];
```

Comment:
the function is `getProgram`, why is the default return value an array?

Reply:
Updated to return an empty object (i.e., `{}`)